tests/language-pronunciation.test

```diff
@@ -36,7 +40,11 @@ b'apka p;'inxpanQ;ist 'atxadz;its; dz.'Orts.tawn m;'is.ts.as v;'Ests;i" \
 Юрась ад'ютант сальюцца ведаю здолею ёю пенсію мною белую памыю ідэю апошнюю змяю,
 якую з'явіўся гэтая неяк якія пояс халуям дыпламатыя ідэя засмяяўся,
 азноб блог сад ксёндз грандж ажыятаж указ,
-бабка пінгпангіст адхадзіць джорджтаўн міжчас везці" "Cyrl"
+бабка пінгпангіст адхадзіць джорджтаўн міжчас везці,
+апдэйт апгрэйд трохбаковы бухгалтар тэхдогляд трохжыльны тэхзона,
+блокбастар бэкграўнд анекдот экзэма шматбаковы артгрупа бортжурнал спіртзавод,
+звышбуйны Ашгабат звышдоўгі звышжорсткі звышзорка айсберг лясгас гасдагавор сельгасзварот касьба,
```

Comment:
And will probably not use soviet acronyms and foreign names as an examples, whenever it is possible

Reply:
fixed